docs/docs-content/automation/palette-cli/commands/vmo.md

```diff
@@ -0,0 +1,69 @@
+
+- [`import-ova`](#import-ova) - Import a vSphere OVA. This subcommand will generate an OVA deployment configuration
+  file. The configuration can then be deployed using the `deploy-ova` subcommand.
+
+- [`migrate-vm`](#migrate-vm) - Migrate one or more VMware vSphere VMs to Palette VMO.
+
+## Prerequisites
+
+- A Healthy VMO cluster. Refer to the [Create a VMO Profile](../../../vm-management/create-vmo-profile.md) for further
+  guidance.
+- One or more VMs hosted in VMware vSphere. The VMs should also operate one the
+  [`virt-v2v` supported guest systems](https://libguestfs.org/virt-v2v-support.1.html).
+
+## Limitations
+
+- You can only use the `vmo` subcommand with VMs hosted in VMware vSphere.
+
+## Deploy OVA
+
+Use the `deploy-ova` subcommand to deploy an imported vSphere OVA to Palette VMO. The following flags are supported by
+the `deploy-ova` subcommand.
+
+| **Short Flag** | **Long Flag**   | **Description**                                                                       | **Type** |
+| -------------- | --------------- | ------------------------------------------------------------------------------------- | -------- |
+| `-f`           | `--config-file` | Specifies an OVA configuration file.                                                  | string   |
+| `-o`           | `--config-only` | Update the OVA configuration file only, without proceeding with the deployment.       | boolean  |
+| `-s`           | `--silent`      | Perform a silent OVA deployment. This flag requires the `--config-file` be specified. | boolean  |
+| `-h`           | `--help`        | Help for the `deploy-ova` subcommand.                                                 | -        |
+
+## Import OVA
+
+Use the `import-ova` subcommand to import a vSphere OVA to Palette VMO. The following flags are supported by the
+`import-ova` subcommand.
+
+| **Short Flag** | **Long Flag**    | **Description**                                                               | **Type** |
+| -------------- | ---------------- | ----------------------------------------------------------------------------- | -------- |
+| `-f`           | `--config-file`  | Specifies an OVA configuration file.                                          | string   |
+| `-o`           | `--config-only`  | Generate the OVA configuration file only, without proceeding with the import. | boolean  |
+|                | `--skip-convert` | Skip OVA conversion.                                                          | boolean  |
+|                | `--skip-image`   | Skip VM image upload.                                                         | boolean  |
+| `-h`           | `--help`         | Help for the `deploy-ova` subcommand.                                         | -        |
+
+## Migrate VM
+
+Use the `migrate-vm` subcommand to migrate one or more VMs from VMware vSphere to Palette VMO. The following flags are
+supported by the `migrate-vm` subcommand.
```

Comment:
Might be worth adding the following context from the CLI's help:

> The migration consists of two phases:
> 1. Disk transfer: all guest disk(s) are transferred to PVs in K8s using CDI + VDDK.
> 2. OS conversion: guest OS on root disk is made bootable on KVM and virtio drivers are installed using virt-v2v.

Reply:
Reworded this a bit and added it.